com.gitlab.Murmele.UDPLogger.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: com.gitlab.Murmele.UDPLogger
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: UDPLogger
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --filesystem=home
+- --share=network
+
+modules:
+# netcdf requires that hdf5 is build with zlib
+- name: zlib
+  buildsystem: cmake-ninja
+  builddir: true
+  sources:
+  - type: git
+    url: https://github.com/madler/zlib.git
+    tag: v1.2.12
+
+# needed by netcdf
+# z lib support needed by netcdf
+- name: hdf5
+# TODO: switch to cmake again
+# For some reason DHDF5_ENABLE_Z_LIB_SUPPORT is not applied when using cmake-ninja instead of simple and the build-commands.
+  buildsystem: simple # cmake-ninja
+  builddir: true
+  build-options:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DHDF5_ENABLE_Z_LIB_SUPPORT:BOOL=ON
+  build-commands:
```

Comment:
because its `config-opts` to pass arguments to CMake.

Reply:
use `cmake-ninja` as buildsystem.

Reply:
done

Reply:
thank you!